app/assets/json/translations/locale-en.json

```
@@ -1040,8 +1040,14 @@
             "get_help": {
                 "other_groups_may_qualify": "Others, like San Francisco teachers and first responders, may qualify for loans.",
                 "some_people_may_qualify": "Some people may qualify for downpayment assistance.",
-                "title": "Get help",
-                "view_full_list_of_programs": "View the full list of programs"
+                "title": "Help",
```

looks like this should still be "Get help"
```suggestion
                "title": "Get help",
```